src/css/ingame_hud/shop.scss

```diff
@@ -241,6 +241,10 @@
 
                             &.complete {
                                 background-color: $colorGreenBright;
+                                
+                                @include DarkThemeOverride {
+                                    background-color: $colorGreenBright;
+                                }
```

Comment:
What is this doing here?

Reply:
DarkThemeOverride for parent (`progressBar` or something like that) for some reason takes over `.complete` background color, even though it should not; so applying color here as well fixes it.